src/twisted/internet/test/test_asyncioreactor.py

```diff
@@ -25,6 +25,8 @@ class AsyncioSelectorReactorTests(ReactorBuilder, SynchronousTestCase):
     """
     L{AsyncioSelectorReactor} tests.
     """
+    @skipIf(not isinstance(get_event_loop(), SelectorEventLoop),
+            "SelectorEventLoop is not default event loop")
```

Comment:
I don't understand this test.

I think that this test should not be skipped but instead set it up in such a way that both twisted and asyncio are using the same event loop.

I guess that the documentaiton for AsyncioSelectorReactor should be updated to explain that if someone want to mix Twisted with asyncio they must do the required setup  for having twisted and asynio on the same event loop.

With the comment that everything is ok on Linux/macOS and that for Windows extra steps are required.

I have never used asyncio so I have no idea how other libraries are setting up their event loop.

maybe have something like https://github.com/tornadoweb/tornado/pull/2686/files

Reply:
I think following Tornado's example is the way to go.

Reply:
I've implemented this